Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:16-alpine as BUILD_IMAGE
+RUN apk add --update --no-cache \
+    make \
+    g++ \
+    jpeg-dev \
+    cairo-dev \
+    giflib-dev \
+    pango-dev
+
+    
+WORKDIR /usr/src/app
+
+COPY . .
+RUN yarn install
+RUN yarn build
+
+FROM node:16-alpine
+
+WORKDIR /usr/src/app
+
+# copy from build image
+COPY --from=BUILD_IMAGE /usr/src/app/build ./build
+COPY --from=BUILD_IMAGE /usr/src/app/node_modules ./node_modules
```

Comment:
After running `yarn build`, you have to run `yarn install --production` in the `./build` folder. Then you can use the production `./build/node_modules` folder instead (it's lighter)